xwiki-platform-core/xwiki-platform-localization/xwiki-platform-localization-rest/src/main/resources/META-INF/components.txt

```diff
@@ -0,0 +1 @@
+org.xwiki.localization.internal.DefaultLocalizationSource
```

Comment:
I'm surprised that Spoon is not complaining in theory you need a blank line at the end of components.txt.

Reply:
I agree Spoon must complain if a blank line is missing at the end of `components.txt`.
This is not the case here, though I agree github's UI is not really explicit about it.
I think a missing trailing blank line is represent by a small icon at the end of the document.
Here for instance: https://github.com/xwiki/xwiki-platform/pull/1595/files#diff-57ce2b01a0829f38517cf869d7b3bfa5260c57a1c2d2f0f52a0c7e3f6ce0af35R104

Reply:
Ah you're right, all good then.